evaluation/human_evaluation/utils/api.py

```diff
@@ -2,11 +2,20 @@
 import os
 from typing import Any
 
-API_URL = os.getenv("CHAT_ENDPOINT", "http://localhost:8000/chatApp")
+API_BASE_URL = os.getenv("CHAT_ENDPOINT", "http://localhost:8000")
 HEADERS = {"accept": "application/json", "Content-Type": "application/json"}
 
+def fetch_endpoints() -> list[str]:
```

Comment:
Are you returning a json? or a list[str]? Also note that your output type needs to handle all cases.

Reply:
It's returning a list[str] but in case it's not able fetch it then it will just return empty list [].

Reply:
Yes but your output type will not pass mypy checks. Try running `python -m mypy --strict evaluation` to make your code mypy compliant.